sdcm/cluster_baremetal.py

```diff
@@ -149,4 +176,5 @@ def __init__(self, **kwargs):
         cluster.BaseMonitorSet.__init__(self,
                                         targets=kwargs["targets"],
                                         params=kwargs["params"])
+        kwargs.pop("targets")
```

Comment:
what's special about targets ? and why do we need to pop it out ?

Reply:
PhysicalMachineCluster -> BaseCluster has no parameter called `targets` and so the constructor raises an exception.